hoomd/md/methods/methods.py

```diff
@@ -708,6 +708,48 @@ def _attach(self):
         super()._attach()
 
 
+class DisplacementCapped(NVE):
+    r"""Modified constant volume, constant energy dynamics with a cap on motion.
```

Comment:
I would describe this as "Newtonian dynamics with a cap on the maximum displacement per time step".